client/app/page.tsx

```diff
@@ -89,18 +84,19 @@ export default function Home() {
                             )}
                         />
 
-                        <div style={{ display: 'flex', justifyContent: 'center' }}>
-                            <Button asChild style={{ marginTop: '10px' }}>
-                                <Link href={isHostLogin ? '/host' : '/competitor'}>Login</Link>
-                            </Button>
+                        <div className="flex justify-center">
+                            <Button className="mt-2 w-[6vw]">Login</Button>
                         </div>
                     </form>
                 </Form>
 
                 {message && <p>{message}</p>}
-                <button className="login-view" onClick={toggleHostLogin}>
-                    {isHostLogin ? 'Login As Competitor' : 'Login As Host'}
-                </button>
+            </div>
+
+            <div className="height-20 flex items-center justify-center">
+                <Button onClick={() => router.push('/leaderboard')} className="w-[6vw]">
+                    Leaderboard
+                </Button>
```

Comment:
This should use a [link](https://ui.shadcn.com/docs/components/button#link), not a click event.

Reply:
This has not been fixed.